### PR TITLE
Use environment variable for analytics key

### PR DESCRIPTION
**Pull request by ryayak1460**
Use GOOGLE_ANALYTICS for the Google Analytics id instead of a
hard-coded, publicly available value everyone can see.  Update the
documentation to reflect this.

Signed-off-by: Ryan Y <ryayak1460@protingumas.com>

### CODE REVIEW

**Comment by BretFisher** [comment]
OK I updated Dockerfile in `develop` branch and Docker Cloud for prod. Ryan you can update docs with the staging code `UA-39303796-11` which anyone can use when dev'ing locally, as a bad GA code may throw something to console.log. Dockerfile is designed to fall back tot that code by default, and I put the prod ID in Docker Cloud.

**Comment by ryayak1460** [comment]
Affects #139.

**Comment by ryayak1460** [comment]
Sure, @BretFisher, on it.

**Comment by ryayak1460** [comment]
Ready for review again.

File: README.md [hunk 1]
```diff
@@ -78,6 +79,7 @@ export OKC_DB_USER=username
 export OKC_DB_PASSWORD=password
 export SMARTY_STREETS_AUTH_ID=smartystreetsauthid
 export SMARTY_STREETS_AUTH_TOKEN=smartystreetsauthtoken
+export GOOGLE_ANALYTICS=UA-39303796-11
```
Comment (wbprice) [inline README.md]: Is this the actual GA code?  We probably shouldn't have this in the README.

Comment (ryayak1460) [inline README.md]: @BretFisher mentioned in #139 that It's okay we use the staging analytics key, just not the production one, and more importantly that it's actually okay if we have this in the readme so contributors can work without possible bugs.

**Pull request merged**